es7-persistence/build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     testImplementation project(':conductor-common').sourceSets.test.output
 
     //Dependencies for signing ES requests with AWS keys
-    compile 'com.amazonaws:aws-java-sdk-s3:1.11.228'
+    compile 'com.amazonaws:aws-java-sdk-s3:1.12.261'
```

Comment:
com.amazonaws:aws-java-sdk-s3:1.12.261 is old (Jul 15, 2022). How about using 1.12.416 (Feb 28, 2023) ?

Reply:
we have 37 usages for 1.12.261 but we have very less usages for 1.1.2.416

Reply:
Using 1.12.261 is fine for now.